lib/scanner.c

```diff
@@ -9,7 +9,7 @@
 #define FLEX_SCANNER
 #define YY_FLEX_MAJOR_VERSION 2
 #define YY_FLEX_MINOR_VERSION 5
-#define YY_FLEX_SUBMINOR_VERSION 39
+#define YY_FLEX_SUBMINOR_VERSION 35
```

Comment:
No reason besides that it's what I have installed. If you accept this patch, you could regenerate that file with your version before commiting.